charts/enterprise-metrics/values.yaml

```diff
@@ -56,7 +56,10 @@ tokengenJob:
   annotations: {}
 
 config:
-  auth_enabled: false
+  auth_enabled: true
```

Comment:
I like the explicit nature of this change, at least in the context of this patch thanks for including it even though it is the GEM default.